node.hcl

```diff
@@ -12,14 +12,23 @@ version "12.18.3" {}
 version "14.16.0" "14.17.0" {}
 version "15.10.0" {}
 
-version "16.1.0" "16.2.0" "16.3.0" {
+version "16.1.0" "16.2.0" "16.3.0" "16.4.0" {
   darwin {
     arch = "arm64"
     source = "https://nodejs.org/dist/v${version}/node-v${version}-${os}-arm64.tar.gz"
   }
+
+  auto-version {
+    github-release = "nodejs/node"
```

Comment:
does this need a [version-pattern](https://cashapp.github.io/hermit/packaging/schema/auto-version/) to convert `v16.4.2` to `16.4.2`?

Reply:
Nah it works with that by default.

Did you test this? Run `hermit manifest auto-version node.hcl`. I think it picks up 14.x.

Reply:
thanks, that was helpful to confirm. I'm guessing it's targeting "[latest](https://github.com/nodejs/node/releases/latest)" which still maps to v14. I'm fine manually updating the v16 channel as needed 👍🏼